ci/playbooks/deploy_podified_ctlplane.yaml

```diff
@@ -0,0 +1,59 @@
+---
+- hosts: controller
+  tasks:
+    - name: Install Ansible, Make, Tar
+      become: true
+      ansible.builtin.package:
+        name:
+          - ansible-core
+          - make
+          - tar
+        state: latest
+
+    - name: Install devsetup dependencies
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/devsetup"
```

Comment:
Can we have a variable? Such as:
`install_yamls_basedir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"`

That would allow an easy use of this kind of role outside of CI.

Reply:
Done